course_discovery/apps/edly_extensions/api/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import include, url
+
+from edly_extensions.api import views
+
+
+app_name = 'api'
+urlpatterns = [
+    url(r'^edly_site_partner/', views.EdlySiteViewSet.as_view(), name='edly_site_partner'),
```

Comment:
Please add the `v1` urls format like other edly apps and modify the URL pattern:
`edly_api/v1/edly_sites/`

Reply:
Done